optuna/samplers/grid.py

```diff
@@ -95,6 +95,10 @@ def __init__(self, search_space):
         self._param_names = sorted(search_space.keys())
         self._n_min_trials = len(self._all_grids)
 
+    def reseed_rng(self) -> None:
```

Comment:
[nits] I think we can remove this code since it is the same behavior as the default implementation.

Reply:
You're absolutely right. Fixed: https://github.com/optuna/optuna/pull/968/commits/359c9ae5fad5c6bf13b8cb38cf206f41d894d71c.